### ISSUE TITLE
Execution time is reported as 0

### ISSUE THREAD

**Issue by jonasbn** [jonasbn/perl-test-timer#13]
The improved diagnostics introduced with release 1.00 and 2.00, report running/execution time as 0. This seems error prone.

**Comment by jonasbn** [comment]
This is issue has been addressed in release 2.01

**Comment by jonasbn** [comment]
The example from @nigelhorne listed in issue #12 demonstrates the bug
```
t/pod-cm.t .... skipped: Test::Pod::Spelling::CommonMistakes required for testing POD spelling
t/pod.t ....... skipped: Author tests not required for installation
# This will take some time because of sleeps

#   Failed test 'should be throttled to 2 seconds, not 10'
#   at t/request.t line 36.
# Test ran 0 seconds and did not execute within specified interval 1 - 6 seconds
# Looks like you failed 1 test of 14.
t/request.t ... 
```

See the diagnostic message:

`Test ran 0 seconds and did not execute within specified interval 1 - 6 seconds`

The `0` is the default value and not the exact timed execution time
